test/battle/hold_effect/eject_pack.c

```diff
@@ -63,3 +63,27 @@ SINGLE_BATTLE_TEST("Eject Pack is triggered by self-inflicting stat decreases")
         ANIMATION(ANIM_TYPE_MOVE, MOVE_CELEBRATE, opponent);
     }
 }
+
+DOUBLE_BATTLE_TEST("Eject Pack will activate before Eject Button since player Wobbuffet is the fastest mon and commander activation will follow")
```

Comment:
I don't quite understand what this test is supposed to do as the name of it doesn't actually match what is being tested.

Reply:
This  could be more of a general test that doesn't involve Commander but it is a fix for #4912. See Commit https://github.com/rh-hideout/pokeemerald-expansion/pull/5246/commits/42edcec59815aac7448458b87b7481d90144cd9e

Reply:
The only issue I have with the test is the name not representing what actually happens in the test

Reply:
```
Eject Pack activation will not trigger an attack from the incoming mon
```
Is this somewhat better?

Reply:
I just realized that this is supposed to be an Eject Pack test, but Eject Button is the only thing that actually activates in this test.